views/my_packages.package.dt

```diff
@@ -112,3 +127,23 @@ block body
 	.inputForm
 		form(method="POST", action="#{req.rootDir}my_packages/#{packageName}/remove")
 			button.danger(type="submit") Remove this package
+
+	:javascript
+		function updateLogoPreview(input) {
+			var f = input.files[0];
+			if (!f) return;
+			document.querySelector(".logoError").textContent = "";
+			if (f.type != "image/png" && f.type != "image/gif" && f.type != "image/jpeg" && f.type != "image/bmp")
+				document.querySelector(".logoError").textContent += "Warning: Invalid image type. ";
```

Comment:
Oh I meant to use `return` to avoid creating the object url and displaying something that's invalid.

Reply:
I used this because it signals the user that the image is now broken instead of showing the valid previous image.